third_party/d3tip/d3_tip.go

```diff
@@ -0,0 +1,330 @@
+// Tooltips for d3.js visualizations
+// https://github.com/Caged/d3-tip
+// Version 0.7.1
+// See LICENSE file for license details
+
+package d3tip
+
+// Source returns the d3-tip.js file
+const Source = `
+{{define "d3tipscript"}}
```

Comment:
How difficult it is to push out the template directive out of these files? This is a nit, I am just trying to make all third party files consistent here - like svgpan they should only contain the JS source as is, without the additional directives so that they are plain simple and once updated to a newer version can be easily upgraded without accidentally losing any content.

Reply:
Should be simple, just move the directives. It's being called only in one place. Will check it later tonight.